code/src/components/Answer.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable react/jsx-closing-tag-location */
+import React from 'react';
+import Equipments from './Equipments';
+
+export const Answer = (props) => {
+  const handleSupport = (event) => {
+    props.setSupport(event.target.value)
+  }
+  return (
+    <>
+      {props.questionNumber === 1
```

Comment:
Another way could be to split all the answers in seperate components since Answer.js contains alot more code this way